doc/specs/stdlib_stats.md

```diff
@@ -166,6 +178,75 @@ program demo_mean
 end program demo_mean
 ```
 
+## `median` - median of array elements
+
+### Status
+
+Experimental
+
+### Description
+
+Returns the median of all the elements of `array`, or of the elements of `array` along dimension `dim` if provided, and if the corresponding element in `mask` is `true`.
+
+After that the elements are sorted in an increasing order, e.g. `array_sorted =
+sort(array)`, the median of the elements of `array` are defined as, if
```

Comment:
suggest to edit as: ... the median of the elements of `array` are defined in the usual way. If 
`n = size(array)` is an even number the median is:
... equation here
and if `n` is an odd number, the median is:
... equation here